examples/pm2-typescript/pages/index.vue

```diff
@@ -0,0 +1,14 @@
+<template>
+  <h1>
+    Hello world !
+  </h1>
+</template>
+
+<script lang="ts">
+import { Component, Vue } from 'vue-property-decorator'
+
+@Component({
+  components: {}
```

Comment:
Just a minimal remark but you can simply stick with `@Component({})` or `@Component` if there is no options to set.